overlays/bootstrap.nix

```diff
@@ -227,7 +228,7 @@ in {
                 ++ final.lib.optional (versionAtLeast "8.10"   && versionLessThan "9.0" && final.stdenv.targetPlatform.isAarch64) ./patches/ghc/ghc-8.10-aarch64-handle-none-rela.patch
                 ++ final.lib.optional (versionAtLeast "9.0"                             && final.stdenv.targetPlatform.isAarch64) ./patches/ghc/ghc-9.0-better-symbol-addr-debug.patch
                 ++ final.lib.optional (versionAtLeast "9.0"                             && final.stdenv.targetPlatform.isAarch64) ./patches/ghc/ghc-9.0-aarch64-handle-none-rela.patch
-
+                # ++ final.lib.optionals (final.stdenv.targetPlatform.isWindows) (fromUntil "8.10"   "9.8" ./patches/ghc-8.10-debug-windows.patch)
```

Comment:
I presume this patch added some debug code for testing.  Should we include the patch in the repo in case we need it again?

Reply:
I think we should just remove it. Depending on the debugging needs, we'll need to come up with a different patch the next time anyway.